database_api.py

```diff
@@ -48,11 +51,60 @@ def get_all_test(self):
             ret = None
         return ret
 
+    def get_course_info(self, course_id, semester):
+        """
+        Retrieves course info from the database for a specific
+        course_id, from a given semester
+
+        Semester is of format Spring/Summer/Fall Year
+
+        This information includes all course evaluation data across
+        different terms for this course_id
+
+        params: query = {
+                            "course_id": "the course_id",
+                            "semester = "semester_in_words"
+                        }
+
+        Example from COS 126, Fall 2021:
+            query = { "course_id": "002051", "semester": "Fall 2021" }
+        """
```

Comment:
Could you also specify what the format of the returned value is? I can find it by running it, but it's good to document the format. 
Am I understanding correctly that the result is a list where the first element (position 0) is the information on the course, and the other elements of the list are evaluations? If so, is there possibly a neater way to format it? Perhaps as a dictionary with one key for info and the other key for evals? What do you think?